pkg/network/ifaceutil/interface.go

```diff
@@ -179,6 +217,40 @@ func MakeVethPair(name, master string, mtu int) (netlink.Link, error) {
 	return veth2, nil
 }
 
+// moveLinkToNamespace moves the veth link to the specified namespace and renames it
+func moveLinkToNamespace(veth netlink.Link, tmpName, finalName string, netNs ns.NetNS, peerLink netlink.Link) (netlink.Link, error) {
```

Comment:
i suggest we make tmpName > name the underlying method shouldn't know about tmpName I think
and finalName >  renameTo which would be a pointer because it maybe nil if we need to just move it to the name space